operate/qa/integration-tests/src/test/java/io/camunda/operate/schema/IndexOldSchemaValidatorIT.java

```diff
@@ -256,7 +256,16 @@ private void whenDatabaseClientReturnsIndexNames(
   }
 
   private Set<String> versionsOf(final IndexDescriptor index, final Set<String> versions) {
-    return new HashSet<>(map(versions, version -> getFullQualifiedIndexName(index, version)));
+    return map(
+            versions,
+            version ->
+                Set.of(
+                    getFullQualifiedIndexName(index, version),
+                    // also test archived indices with underscore in suffix
+                    getFullQualifiedIndexName(index, version) + "2025-02-11_20"))
```

Comment:
:wrench: Extending the test coverage for all tests in this class is nice but I'd suggest adding one dedicated test method that specifically tests various rollover patterns.